test/WebJobs.Script.Tests.Integration/Host/ScriptHostManagerTests.cs

```diff
@@ -32,7 +32,7 @@ public ScriptHostManagerTests()
         public async Task UpdateFileAndRestart()
         {
             CancellationTokenSource cts = new CancellationTokenSource();
-            var fixture = new NodeEndToEndTests.TestFixture();
+            var fixture = new CSharpEndToEndTests.TestFixture();
```

Comment:
@ahmelsayed Please revert the changes to this file before checking in - these tests are intentionally using a Node host. If needed you can just conditional out the file based on the MONO flag in the test proj file.